templates/cis.rules.erb

```diff
@@ -12,7 +12,6 @@
 
 # Feel free to add below this line. See auditctl man page
 
-
 # CIS Benchmark Adjustments
```

Comment:
This cookbook is intended to focus on general implementation. Tie it to CIS is not a good idea.